copier.yml

```diff
@@ -77,7 +77,7 @@ documentation:
   type: str
   help: "Do you want to include documentation for your project and which framework do you want to use?"
   choices:
-    "Sphinx (https://www.pyopensci.org/pyos-sphinx-theme)": sphinx
+    "Sphinx (https://pydata-sphinx-theme.readthedocs.io/en/stable/index.html)": sphinx
```

Comment:
This is just a tiny change - pyos-sphinx-theme is specific to our guidebooks. so let's go with the community lead pydata-sphinx-theme instead 🚀  :)